.github/workflows/build.yml

```diff
@@ -1,15 +1,15 @@
 on:
   push:
     branches:    
-      - master
+      - main
 name: Build
 jobs:
   macOS:
     name: macOS platform
     runs-on: macOS-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@main
```

Comment:
No no 😛  This is the branch of the [`action/checkout`](https://github.com/actions/checkout/releases). BTW, we should use a release number instead of `master`. V2.3.1 should do it. Thanks.

Reply:
Ah, thanks. I'll take a look and push a commit to fix that.

Reply:
https://github.com/ably/delta-codec-cocoa/pull/11/commits/a9d197826881d96b81edce4e95a9f27b2785c31d

Reply:
Perfect, thanks.